### PR TITLE
Parameterize ceph monitor hosts from reservation file

### LINKED ISSUE #119: Parameterize ceph monitors host
I perform my tests at Nantes since Rennes vlan are dead. To make deployment faster I wanna use Nantes' Ceph, but monitors host are fixed in [`roles/registry/file/ceph.conf`](https://github.com/BeyondTheClouds/enos/blob/6876c892f76617e46d5952a0a5ebbbf93080239c/enos/ansible/roles/registry/files/ceph.conf#L2). I rather propose to parameterize this into the `reservation.yml`.

### PR DESCRIPTION
**Pull request by rcherrueau**
Fix #119

### CODE REVIEW

**Comment by rcherrueau** [comment]
OK, let me write something in the docmuentation :)

**Comment by rcherrueau** [comment]
Done! I put this into the customization part.

File: docs/customization/index.rst [hunk 1]
```diff
@@ -91,6 +91,39 @@ Ansible configuration
...
+      ceph: true|false
+      ceph_keyring: path to your keyring
+      ceph_id: your ceph id
+      ceph_rbd: rbd in the form "pool/rbd"
+      ceph_mon_host: list of ceph monitor addresses
+
+
+* ``ceph: false`` starts a fresh registry that caches the images for
+  the duration of the experiment.
+* ``ceph: true`` uses as registry the existing ``ceph_rbd`` Ceph Rados
```
Comment (msimonin) [inline docs/customization/index.rst]: uses a registry whose backend is the existing `ceph_rbd`

File: docs/customization/index.rst [hunk 2]
```diff
@@ -91,6 +91,39 @@ Ansible configuration
...
 configuration file, the ``ANSIBLE_CONFIG`` environment variable can be used.
-Further information can be found : `see here 
+Further information can be found : `see here
 <http://docs.ansible.com/ansible/intro_configuration.html>`_.
 
+
+Using a persistent registry with Ceph
+-------------------------------------
+
+To save time during deployment, you can setup a Ceph Rados Block
```
Comment (msimonin) [inline docs/customization/index.rst]: Enos deploys a fresh registry that acts as a private docker registry mirroring the official one and cache containers close to your deployment resources.

To get a persistent registry you can use  a persistent Ceph Rados Block Device for the registry backend. Image will be cached during the first deployment and reused for the subsequent deployments

The relevant configuration ...

Comment (rcherrueau) [inline docs/customization/index.rst]: Done

File: docs/customization/index.rst (hunk 1 above)
Comment (rcherrueau) [inline docs/customization/index.rst]: Done

**Pull request merged**